src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -213,10 +223,13 @@ private static void ValidateBlobResponse(Response<DownloadRegistryBlobResult> bl
 
         private static async Task<Stream> ProcessLayer(ContainerRegistryContentClient client, OciDescriptor layer)
         {
-            if (!string.Equals(layer.MediaType, BicepMediaTypes.BicepModuleLayerV1Json, MediaTypeComparison))
+            string[] allowedMediaTypes = { BicepMediaTypes.BicepModuleLayerV1Json, BicepMediaTypes.BicepProviderArtifactLayerV1TarGzip };
+            // media types are case insensitive
+            if (!allowedMediaTypes.Contains(layer.MediaType, StringComparer.OrdinalIgnoreCase))
             {
-                throw new InvalidModuleException($"Did not expect layer media type \"{layer.MediaType}\".", InvalidModuleExceptionKind.WrongModuleLayerMediaType);
+                new InvalidModuleException($"Did not expect layer media type \"{layer.MediaType}\".", InvalidModuleExceptionKind.WrongModuleLayerMediaType);
             }
+          
```

Comment:
this section is made to special case handle an  oci artifact that is not a module artifact, I will refactor this code to decouple the handling of oci artifacts form module specific validation so we can reuse this logic in separate PRs